src/shellbar.scss

```diff
@@ -229,5 +289,64 @@ $block: #{$fd-namespace}-shellbar;
         display: inline-block;
       }
     }
+
+    .#{$block}__counter--notification {
+      max-width: 2.5rem;
+      text-overflow: ellipsis;
+      white-space: nowrap;
+      position: absolute;
+      background: $fd-content-badge-background;
+      border: $fd-content-badge-background;
+      color: $fd-shellbar-color;
+      line-height: 0.8;
+      top: 10px;
+      max-height: 16px;
+    }
+
+    .#{$block}__input-group__addon {
+      background-color: $fd-shellbar-background-color;
+    }
+
+    .#{$block}__input-group {
+      background-color: $fd-shellbar-background-color;
+      cursor: text;
+
+      &:hover {
+        border-color: $fd-field-border-color;
+      }
+
+      .#{$block}__input-group__input {
+        border: none;
+        color: $fd-shellbar-color;
+
+        &:hover {
+          background-color: $fd-shell-hover-background;
+        }
+      }
+    }
+  }
+
+  &__button--menu {
+    &::after {
```

Comment:
The font-size of the Menu Button is not correct. This is an issue not related to this PR, it comes from the Button component where for some reason the correct font-size is overwritten to 14px. It should be @sapMFontMediumSize which is .875rem. If you want you can specify the correct font-size here. It should follow the Button component specs